README.md

```diff
@@ -11,13 +11,13 @@ PublicLab.org
 [![Newcomers welcome](https://img.shields.io/badge/newcomers-welcome-pink.svg)](https://code.publiclab.org) [![GitHub license](https://img.shields.io/github/license/publiclab/plots2?logo=gpl)](https://github.com/publiclab/plots2/blob/main/LICENSE)
 [![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/publiclab/plots2/)
 
-The content management system for the Public Lab research community, the `plots2` web application is a combination of a group research blog of what we call "research notes" and a wiki. Read more about the [data model here](https://github.com/publiclab/plots2/blob/main/doc/DATA_MODEL.md).
+The content management system for the Public Lab research community, the `plots2` web application is a combination of a group research blog -what we call "research notes"-and a wiki. Read more about the [data model here](https://github.com/publiclab/plots2/blob/main/doc/DATA_MODEL.md).
```

Comment:
Here I treated the "what we call 'research notes'" as a sort of aside because its purpose is to add more information to/define what a "group research blog" is.  If we remove that clause the sentence would still make sense so the dashes were used to represent that its "extra" context/information. 

However, a comma could also work in place of the dashes as well. It's more or less the written equivalent of speaking to someone about something but quickly going off-topic to define or add more context to something you said before continuing with the main topic.

Reply:
Ok got it

Reply:
I think using colons would be better in this case. What so you think?

Reply:
Yes, colons make a lot more sense in the current form, but he changed the sentence form so they can be removed.
Furthermore, using "what we call" in colons does not work well.

Reply:
oh yeah I see...cool lets leave it as is then. Thanks